src/Slub/Domain/GitHub/Properties/Repository.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
I'd like the domain to remain VCS agnostic. I recommend adding VCS specific objects as separate DTOs within the adapters.